src/main/java/duke/ui/Ui.java

```diff
@@ -293,9 +293,7 @@ public void showInsufficientBalance(Payment payment) {
     }
 
     public void showSuccess(String type, String value) {
-        showLine();
         System.out.println(UI_PADDING + "Your " + type + " has been updated to " + value + ".");
```

Comment:
Can we standardize all calls to ` System.out.println(UI_PADDING + message);` in Ui.java to be replaced with `showMessage(message)`